setup.py

```diff
@@ -25,6 +25,7 @@
 setup(
     name="financeager",
     version=__version__,
```

Comment:
I think this can be omitted then, including the corresponding import.

Reply:
Ok, it's done